.env.example

```diff
@@ -1,6 +1,6 @@
 # django
 APP_ENV='local' # local, staging, production
-SECRET_KEY='' # generate from https://djecrety.ir/
+SECRET_KEY='i%&w001-tkaz_f&r#=1%9^90eaqdhu1byjjx#11n*zf&ua@s9@' # generate from https://djecrety.ir/
```

Comment:
I think you forgot to remove this value.
This secret key's value should not be present in `.env.example` file. This file is supposed to be just an example. Everyone's secret key would be different and stored in their personal `.env` file